docs/anchoring-assets/anchor-platform/index.mdx

```diff
@@ -11,14 +11,14 @@ The Anchor Platform accomplishes this by implementing the ecosystem's standardiz
 
 Below is a list of SEPs currently supported:
 
-- [SEP-1][sep-1]: Stellar Info File
-- [SEP-10][sep-10]: Stellar Authentication
+- [SEP-1][sep-1]: [Stellar Info File][sep1-ap]
+- [SEP-10][sep-10]: [Stellar Authentication][sep10-ap]
 - [SEP-12][sep-12]: KYC API
```

Comment:
There aren't sections on SEP 12, 31, or 38 yet, right? Should they be included here?

Reply:
No, they aren't written yet. I think they should be there to indicate that functionality is ready